bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/GF_HHH_benchmarks/GF_HHH_c3_0_d4_0/GF_HHH_c3_0_d4_0_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+set group_subprocesses Auto 
+set ignore_six_quark_processes False 
+set loop_optimized_output True 
+set complex_mass_scheme False 
+import model loop_sm_c3d4
+define lep = e+ e- mu+ mu-
+define nus = ve~ vm~ ve vm
```

Comment:
You can remove these lines if not used anywhere else

Reply:
Thanks, I've removed them.